wifiphisher/pywifiphisher.py

```diff
@@ -158,6 +170,21 @@ def parse_args():
         help="Force the usage of hostapd installed in the system",
         action='store_true')
 
+    parser.add_argument("-dP",
+                        "--data_path",
```

Comment:
The "data_path" as a name indicates the location of the whole "data" directory, not only the "phishing_pages subdirectory.

I propose to change this to `-pPD` or `--phishing-pages-directory`.